frontend/src/components/slides/slides-component.tsx

```diff
@@ -14,12 +14,6 @@ import { useEventListener } from "@/hooks/useEventListener";
 import { cn } from "@/utils/cn";
 
 import "./slides.css";
-import "swiper/css";
-import "swiper/css/virtual";
-import "swiper/css/keyboard";
-import "swiper/css/navigation";
-import "swiper/css/pagination";
-import "swiper/css/scrollbar";
```

Comment:
These CSS imports quietly worked before because we had `allowJs: true`, which let TypeScript accept non-TypeScript imports in TS files. 

Moving them to the CSS file makes this behavior explicit and allows us to safely disable `allowJs` across the reset of the code base.